pymongo/asynchronous/command_cursor.py

```diff
@@ -382,6 +382,12 @@ async def __aexit__(self, exc_type: Any, exc_val: Any, exc_tb: Any) -> None:
         await self.close()
 
     async def to_list(self) -> list[_DocumentType]:
+        """Converts the contents of this cursor to a list.
+
+        If the cursor is empty or has no more results, an empty list will be returned.
```

Comment:
Suggest adding an example and saying this is a more efficient verison of `[doc async for doc in cursor]`.

Reply:
Done. The synchronization to the sync versions will be handled by https://github.com/mongodb/mongo-python-driver/pull/1756.